node/core/approval-voting/src/lib.rs

```diff
@@ -100,7 +101,8 @@ const APPROVAL_CACHE_SIZE: usize = 1024;
 const TICK_TOO_FAR_IN_FUTURE: Tick = 20; // 10 seconds.
 const LOG_TARGET: &str = "parachain::approval-voting";
 const DEBUG_LOG_TARGET: &str = "parachain::ladi-debug-approval-voting";
-const MALICIOUS_FREQUENCY: usize = 5_000;
+const MALICIOUS_MEAN: f64 = 2_500f64;
+const MALICIOUS_SDEV: f64 = 70f64;
```

Comment:
nit: `.._STDDEV` is the common abbreviation.

Reply:
Yeah, I wanted MEAN and SDEV to have the same length :)